neo/rawio/intanrawio.py

```diff
@@ -104,8 +104,8 @@ def _parse_header(self):
         # signals
         signal_channels = []
         for c, chan_info in enumerate(self._ordered_channels):
-            name = chan_info["native_channel_name"]
-            chan_id = str(c)  # the chan_id have no meaning in intan
+            name = chan_info["custom_channel_name"]
+            chan_id = chan_info["native_channel_name"]
```

Comment:
Should we call this channel_id so that we have the same name here as later in the code so we don't call this chan_id here and channel_id later in the code?

Reply:
I would prefer this.